src/efi.rs

```diff
@@ -14,6 +14,7 @@ use anyhow::{bail, Context, Result};
 use openat_ext::OpenatDirExt;
 
 use chrono::prelude::*;
+use structopt::lazy_static::lazy_static;
```

Comment:
We should add the dependency to `Cargo.toml` directly; it looks like a bug that structopt is re-exporting it.